docs/config/config.xml

```diff
@@ -70,37 +70,10 @@
                 <test value="InstrumentName">Instrument Display Name</test>
             </CertificationInstruments>
         </Certification>
-
     </study>
     <!-- end of study definition -->
 
     <gui>
         <showDatabaseQueries>0</showDatabaseQueries>
     </gui>
-
```

Comment:
should probably do the same to the test config ?

Reply:
okay, once the PR that's blocking this (#8290) is in and I can rebase one this I'll do that.